tests/test_util/aiohttp.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, method, url, status, response):
 
     def match_request(self, method, url):
         """Test if response answers request."""
-        return method == self.method and url == self.url
+        return method == self.method and url.startswith(self.url)
```

Comment:
That doesn't seem to be the case, [see the source](http://bazaar.launchpad.net/~jamielennox/requests-mock/master/view/head:/requests_mock/adapter.py#L128). The difference however is that the query string was tested separately.